nomad/client_csi_endpoint.go

```diff
@@ -42,7 +43,11 @@ func (a *ClientCSIController) AttachVolume(args *cstructs.ClientCSIControllerAtt
 	}
 
 	// Make the RPC
-	return NodeRpc(state.Session, "CSIController.AttachVolume", args, reply)
+	err = NodeRpc(state.Session, "CSIController.AttachVolume", args, reply)
+	if err != nil {
+		return fmt.Errorf("attach volume: %v", err)
```

Comment:
Does our logger have a way of attaching the RPC name to the output so that we don't have to do this sort of thing every time we bubble-up a log?